homeassistant/helpers/entity_component.py

```diff
@@ -364,11 +364,12 @@ def _update_entity_states(self, now):
 
         This method must be run in the event loop.
         """
-        if self._process_updates:
+        if self._process_updates.locked():
+            self.component.logger.warning(
+                'Slow update circle on %d', self.component.domain)
```

Comment:
It should be `%s` for strings. %d is for numbers.

Let's change the text to be more explanatory:

```python
self.component.logger.warning(
    "Updating %s took longer than the scheduled update interval %s",
    self.component.domain, self.scan_interval)
```

Would actually be better if we could call out the platform name (we should pass that into the constructor)